src/components/stepforms/widgets/List.vue

```diff
@@ -96,6 +96,7 @@ export default class ListWidget extends Mixins(FormWidget) {
       valueCopy.push(this.defaultChildValue);
     }
     return valueCopy.map(value => ({
+      uid: this.getUniqueId(value),
```

Comment:
I don't think it's a good idea to put that in a computed property: at each evaluation loop, the id will change, triggering a destruction of the corresponding element.

Reply:
Seeing the implementation of the function below, I understand that this is not an id that will change each time (like lodash's `uniqueId` but a deterministic computation from the row's values.
This also has drawbacks : at each modification of the row's value by the user (e.g. typing a letter in a text field), the whole row will be destroyed and recreated.